pubsub/iam.js

```diff
@@ -0,0 +1,183 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var async = require('async');
+var utils = require('./subscription');
+var createTopic = utils.createTopic;
+var subscribe = utils.subscribe;
+
+// [START get_topic_policy]
+function getTopicPolicy(topic, callback) {
+  // Retrieve the IAM policy for the provided topic
+  topic.iam.getPolicy(callback);
+}
+// [END get_topic_policy]
+
+// [START get_subscription_policy]
+function getSubscriptionPolicy(subscription, callback) {
+  // Retrieve the IAM policy for the provided subscription
+  subscription.iam.getPolicy(callback);
```

Comment:
Hmm.. this seems weird too. What are you actually showing here? I think the 'meaty' bit is what's passed into callback. Can you find a way to show both the call and the callback at once?